arm-servicebus/2015-08-01/swagger/servicebus.json

```diff
@@ -2461,67 +2423,71 @@
             "Unknown"
           ],
           "x-ms-enum": {
-            "name": "AvailabilityStatus",
+            "name": "EntityAvailabilityStatus",
```

Comment:
Rather than repeatedly defining the same enum at multiple places, this can be defined in the definitions section once as an enum and then referenced wherever required. This reduces duplication and chances of errors.